packages/gatsby-plugin-gatsby-cloud/src/components/Indicator.js

```diff
@@ -81,7 +153,29 @@ const Indicator = () => {
       buildId !== newBuildInfo?.latestBuild?.id &&
       currentBuild?.buildStatus === BuildStatus.SUCCESS
     ) {
-      setBuildInfo({ ...newBuildInfo, buildStatus: BuildStatus.SUCCESS })
+      if (refreshNeeded) {
```

Comment:
Where does `refreshNeeded` come from? I only see it being initialized to `false` on 34 and then not used again